src/MDButton.jsx

```diff
@@ -40,7 +41,7 @@ class MDButton extends Component {
 }
 
 MDButton.propTypes = {
-  children: PropTypes.object,
+  children: PropTypes.any,
```

Comment:
It can be PropTypes.element or PropTypes.text.. 
Maybe I should use PropTypes.oneOf([PropTypes.element, PropTypes.text])